<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IF-SC WeldWay</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_18">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="findZeroPushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Buscar Zeros</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="initPosButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Mover Posição Inicial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="finalPosButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Mover Posição Final</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="correctButton">
      <property name="text">
       <string>Corrigir</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="5">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="executeButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Executar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="statusTab">
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>170</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Eixo X:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="xvalueLabel">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Eixo Y:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="yvalueLabel">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Eixo Z:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="zvalueLabel">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Eixo A:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="avalueLabel">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Eixo B:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="bvalueLabel">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="getValuesButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Get</string>
        </property>
       </widget>
       <widget class="QLabel" name="anotherLabel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>250</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="emergencyLabel">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>251</width>
          <height>171</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Emergência</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="trajectoryTab">
       <attribute name="title">
        <string>Trajetória</string>
       </attribute>
       <layout class="QHBoxLayout" name="trajectoryTabLayout"/>
      </widget>
      <widget class="QWidget" name="positonTab">
       <attribute name="title">
        <string>Movimentação</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>14</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Eixo Y:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="yposUnitComboBox">
            <item>
             <property name="text">
              <string>mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>m</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>pulsos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&quot;</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="ypositionSpinBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QSlider" name="ypositionSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="zpositionSpinBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QSlider" name="zpositionSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QComboBox" name="zposUnitComboBox">
            <item>
             <property name="text">
              <string>mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>m</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>pulsos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&quot;</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="xpositionSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="xpositionSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="xposUnitComboBox">
            <item>
             <property name="text">
              <string>mm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>m</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>pulsos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&quot;</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Eixo X:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Eixo Z:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>15</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="vorientationSpinBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QSlider" name="vorientationSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="horientationSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="horientationSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>Orientação vertical:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>Orientação horizontal:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>º</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>º</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_13">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>14</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QSlider" name="speedSlider">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Velocidade:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="speedUnitComboBox">
            <item>
             <property name="text">
              <string>m/min</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mm/s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>rpm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>rps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="speedSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="oscillationTab">
       <attribute name="title">
        <string>Oscilações</string>
       </attribute>
       <layout class="QHBoxLayout" name="oscillationTabLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFerramentas">
    <property name="title">
     <string>Ferramentas</string>
    </property>
    <addaction name="actionConnection"/>
    <addaction name="actionStatus"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuFerramentas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnection">
   <property name="text">
    <string>Conexão</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
